File: TodoList_App/todolist_app/lib/application/dto/auth_output_user_dto.dart
class AuthOutputUserDTO {
  AuthOutputUserDTO(
      {required this.id,
      required this.name,
      required this.email,
      required this.token,
      required this.password});

  String id;
  String name;
  String email;
  String token;
  String password;

  factory AuthOutputUserDTO.fromJson(Map<String, dynamic> json) =>
      AuthOutputUserDTO(
        id: json["id"],
        name: json["name"],
        email: json["email"],
        token: json["token"],
        password: json["password"]
      );
}

File: WebAPI_TodoList/WebAPI_TodoList.Application/DTO/Auth/AuthOutputUserDTO.cs
namespace WebAPI_TodoList.Application.DTO.Auth;

public class AuthOutputUserDTO
{
    public AuthOutputUserDTO()
    {
        Id = string.Empty;
        Name = string.Empty;
        Email = string.Empty;
        Token = string.Empty;
    }

    public string Id { get; set; }
    public string Name { get; set; }
    public string Email { get; set; }
    public string Token { get; set; }
}
